### ISSUE TITLE
Using relative volume paths instead of named volumes

### ISSUE THREAD

**Issue by IcelandicIcecream** [escapingnetwork/core-keeper-dedicated#71]
Hey guys! I remember I was pretty confused with volumes when I first started my server. Like whether it was working or not and how to back them up and etc. I definitely feel like using or at least encouraging relative paths would be better instead of defaulting to named volumes in the readme. So everything is accessible within 1 directory. 

This technically wouldn't break anything I think.

```
services:
  core-keeper:
    container_name: core-keeper-dedicated
    image: escaping/core-keeper-dedicated
    volumes:
      - ./server-files:/home/steam/core-keeper-dedicated ## instead of server-files:...
      - ./server-data:/home/steam/core-keeper-data ## instead of server-data:....
    env_file:
      - ./core.env
    restart: always
    stop_grace_period: 2m
```

Not sure if I'm missing anything or if there was a specific reason for named volumes.

**Comment by Diyagi** [comment]
Using volumes instead of binds was a decision made over the fact that the container did not check or enforced permissions on the files it was using, this would causes all sort of issues going from world not saving to crashes.

This has since been solved in the 2.0 version by enforcing permissions in the startup process and also allowing the user to change the PUID and PGID of the user thats running inside the container.

It should be safe now to use binds, worst case is if you do not set the PUID and PGID correctly you will have some headache trying to edit anything from the container files since it will be under another users permissions.

**Comment by arguser** [comment]
Thank you @Diyagi for addressing the issues!
